startup/gui/ocio.py

```diff
@@ -100,6 +100,7 @@ def __plugSet( plug ) :
 def __displayTransformCreator( name ) :
 
 	result = GafferImage.DisplayTransform()
+	result["channels"].setValue( "*[RGB]" )
```

Comment:
would this apply a display transform to auxillary channels?
I think we dont want to display something like Z or alpha with a transform.